gsa/src/web/components/dashboard/display/datadisplay.js

```diff
@@ -83,6 +80,38 @@ const FilterString = styled.div`
   overflow: hidden;
 `;
 
+const IconBar = styled.div.attrs({
+  className: 'display-icon-bar',
+})`
+  height: 26px;
+  width: 100%;
+  min-height: 26px;
+  display: flex;
+  flex-grow: 1;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 5px 10px;
+  position: absolute;
+  z-index: ${Theme.Layers.higher};
+  transition: opacity 500ms;
+`;
+
+const DisplayBox = styled.div`
+  display: flex;
+  flex-grow: 1;
+  position: relative;
+
+  & .display-icon-bar {
```

Comment:
In styled-components you can directly reference components without the detour with classnames.
So what you can do here, is something like that:
'''
${IconBar}: {
  visibility: hidden;
  opacity: 0;
}